portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,42 @@
   width: 650px;
 }
 
+title {
+    text-align: center;
+}
+
 #greeting-container {
   margin-top: 20px;
 }
+p1 {
```

Comment:
You seem to be using several different fonts: (Times New Roman, Lucida, Gill Sans, Impact, Haettenschweiler, etc). I would advise to be consistent on the font you use.